### PR TITLE
DatePicker 

### PR DESCRIPTION
**Pull request by pkanuganti**
[Story](https://rentpath.leankit.com/card/606922844)

### CODE REVIEW

File: packages/react-ui-core/src/DateInput/DateInput.js [hunk 1]
```diff
@@ -0,0 +1,24 @@
+import React, { Component } from 'react'
+import PropTypes from 'prop-types'
+import DatePicker from 'react-datepicker'
+
+export default class DateInput extends Component {
+
```
Comment (bethstevic) [inline packages/react-ui-core/src/DateInput/DateInput.js]: Do we want to go ahead and set up the `themed` like the new components we have been writing? Something like `@themed(['DateInput'])`

**Comment by bethstevic** [comment]
Looks good 👍

File: packages/react-ui-core/src/DateInput/DateInput.js (hunk 1 above)
Comment (nray621) [reply packages/react-ui-core/src/DateInput/DateInput.js]: +1

Comment (pkanuganti) [reply packages/react-ui-core/src/DateInput/DateInput.js]: Datepicker is an external lib. So all the css class are global . so we don't need to have setup like @themed(['DateInput'])

File: packages/react-ui-core/src/DateInput/DateInput.js [hunk 2]
```diff
@@ -0,0 +1,24 @@
+import React, { Component } from 'react'
```
Comment (nray621) [inline packages/react-ui-core/src/DateInput/DateInput.js]: Isn't this just an empty wrapper that passes props through to the `DatePicker`?
Would something closer to what you wrote for the story's component maybe be better? idk, just throwing that out there

Comment (pkanuganti) [reply packages/react-ui-core/src/DateInput/DateInput.js]: Its just wrapper to pass props to DatePicker. In story book, we have default behaviour for Datepicker.

File: packages/react-ui-core/src/DateInput/DateInput.js (hunk 1 above)
Comment (nray621) [reply packages/react-ui-core/src/DateInput/DateInput.js]: You should still have it imo

**Comment by nray621** [comment]
ok 👍

**Pull request merged**